### ISSUE TITLE
Can this be used to send tracking events to GA4

### ISSUE THREAD

**Issue by eonist** [gaapi4py/gaapi4py#11]
Thanks

**Comment by DeflateAwning** [comment]
No. This library appears to be for Universal Analytics, API v4. Also, it's read-only

**Comment by eonist** [comment]
Aha. Read only. Why is it called GA API 4 if its 3?

**Comment by DeflateAwning** [comment]
Google is awful at naming. As far as I can tell, this library is for Universal Analytics (UA), which is deprecated on 2023-07-01 (evidence: uses View ID, example contains 'UA' in the property ID). UA's final API version was the Analytics API v4. 

The current version of Google Analytics is Google Analytics 4, API v1. Someone inside Google is laughing.

**Comment by eonist** [comment]
Haha. Yeah. Google in general. They would do well breaking up google into smaller startups running autonomously from the mothership. Btw. Adding SDK-less GA4 support for swift before the deadline hits. Im sure they will postpone the deadline a bit. 😅 It can be ported to other languages as well: https://github.com/sentryco/Telemetry Im using this as a part of my observability suite. This and https://github.com/sentryco/Logger
